presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -190,6 +190,46 @@ public void testFiltersAndProjections()
         assertQuery("SELECT if(orderkey % 2 = 0, quantity_by_linenumber) FROM orders_ex");
     }
 
+    @Test
+    public void testAnalyzeStats()
+    {
+        assertUpdate("ANALYZE region", 5);
```

Comment:
What is 5 here ?

Reply:
Added a comment. It is the expected number of rows to be updated after running analyze.